sbndcode/BlipRecoSBND/Utils/BlipUtils.cc

```diff
@@ -155,7 +157,9 @@ namespace BlipUtils {
       float totE = tblip.Energy + pinfo.depEnergy;
       float w1 = tblip.Energy/totE;
       float w2 = pinfo.depEnergy/totE;
-      tblip.Position    = w1*tblip.Position + w2*pinfo.position;
+      tblip.Position.SetXYZ( w1*tblip.Position.X() + w2*pinfo.position.X(), 
+                            w1*tblip.Position.Y() + w2*pinfo.position.Y(),
+                            w1*tblip.Position.Z() + w2*pinfo.position.Z());
```

Comment:
This is the hardest one, being weighted. I would use the [accumulator](https://code-doc.larsoft.org/docs/latest/html/classgeo_1_1vect_1_1MiddlePointAccumulatorDim.html) directly:
```suggestion
      geo::vect::MiddlePointAccumulator mpalg;
      mpalg.add(tblip.Position, w1);
      mpalg.add(pinfo.Position, w2);
      tblip.Position = mpalg.middlePoint();
```
(`#include "larcorealg/Geometry/geo_vector_utils.h"`)